.circleci/config.yml

```diff
@@ -107,6 +109,27 @@ jobs:
       MAKE_JOB_COUNT: 8
       WAF: "<< parameters.waf >>"
       NGINX_VERSION: "<< parameters.nginx-version >>"
+  coverage:
+    environment:
+      DOCKER_BUILDKIT: 1
+    steps:
+    - checkout
+    - run: git submodule sync && git submodule update --init --recursive
+    - run: echo -e "ARCH=amd64\nBASE_IMAGE=nginx:1.26.0\n" > nginx-version-info
+    - run:
+        command: 'make coverage'
+        environment:
+          ARCH: x86_64
+          MAKE_JOB_COUNT: 8
+          BUILD_TYPE: RelWithDebInfo
+          NGINX_VERSION: 1.26.0
+          WAF: ON
+    - codecov/upload:
+        file: .musl-build/coverage.lcov
```

Comment:
Even if you specified as file, codecovecli by default still search for coverage file. You might also want `upload_args: --disable-search`